server/QuestionsAndAnswers/answers.js

```diff
@@ -0,0 +1,20 @@
+var routes = require('express').Router();
+var axios = require('axios');
+var url = require('url');
+var config = require('../../config');
+var endPoint = 'https://app-hrsei-api.herokuapp.com/api/fec2/:hr-atx/qa/answers'
+
+var auth = { headers: { 'Authorization': `${config}` }, params: { product_id: 1 } };
+
+routes.put(`/:answer_id/*`, (req, res) => {
+  var path = url.parse(req.url, true).path.split('/').at(-1);
+  axios.put(endPoint + `/${req.params.answer_id}/${path}`, { answer_id: req.params.answer_id }, auth)
```

Comment:
you may want to drive consistency here with the same convention for string interpolation/concatenation:

```node.js
const [ PATH ] = new url(url).path.split("/")
const ENDPOINT = "https://app-hrsei-api.herokuapp.com/api/fec2/:hr-atx/qa/answers"
const { 
  params: { answer_id: ANSWER_ID },
  url
} = req

const ENDPOINT_WITH_PATH = `${ENDPOINT}/${ANSWER_ID}/${PATH}`

axios.put(
  ENDPOINT_WITH_PATH,
  { answer_id: ANSWER_ID },
  auth
)
```

Reply:
Hmm, the [ PATH ] syntax, seems unfamiliar, reading up on it now. And the consistency is nice. i prefer the way you put it.